specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -0,0 +1,2655 @@
+     {
+      "name": "faceListId",
+      "in": "path",
+      "description": "faceListId of an existing face list. Valid character is letter in lower case or digit or '-' or '_', maximum length is 64.",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "name": "persistedFaceId",
+      "in": "path",
+      "description": "persistedFaceId of an existing face. Valid character is letter in lower case or digit or '-' or '_', maximum length is 64.",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful call returns an empty response body."
+     },
+     "default": {
+      "description": "Error response.",
+      "schema": {
+       "$ref": "#/definitions/APIError"
+      }
+     }
+    },
+    "produces": [
+     "application/json"
+    ]
+   }
+  }
+ },
+ "x-ms-paths": {
+  "/detect?overload=url": {
+   "post": {
+    "description": "Detect human faces in an image and returns face locations, and optionally with faceIds, landmarks, and attributes.",
+    "operationId": "Faces_Detect",
+    "parameters": [
+     {
+      "name": "returnFaceId",
+      "in": "query",
+      "description": "Return faceIds of the detected faces or not. The default value is true. ",
+      "type": "boolean",
+      "default": "true",
+      "enum": [
+       "false",
+       "true"
+      ]
+     },
+     {
+      "name": "returnFaceLandmarks",
+      "in": "query",
+      "description": "Return face landmarks of the detected faces or not. The default value is false. ",
+      "type": "boolean",
+      "default": "false",
+      "enum": [
+       "false",
+       "true"
+      ]
+     },
+     {
+      "name": "returnFaceAttributes",
+      "in": "query",
+      "description": "Analyze and return the one or more specified face attributes in the comma-separated string like \"returnFaceAttributes=age,gender\". Supported face attributes include age, gender, headPose, smile, facialHair, glasses and emotion. Note that each face attribute analysis has additional computational and time cost.",
+      "type": "string"
+     },
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     },
+     {
+      "$ref": "#/parameters/ImageUrl"
+     }
+    ],
+    "consumes": [
+     "application/json"
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful call returns an array of face entries ranked by face rectangle size in descendingorder. An empty response indicates no faces detected. ",
+      "schema": {
+       "$ref": "#/definitions/DetectResponse"
+      },
+      "examples": {
+       "application/json": [
+        {
+         "faceId": "c5c24a82-6845-4031-9d5d-978df9175426",
+         "faceRectangle": {
+          "width": 78,
+          "height": 78,
+          "left": 394,
+          "top": 54
+         },
+         "faceLandmarks": {
+          "pupilLeft": {
+           "x": 412.7,
```

Comment:
Note that you have a type mismatch here!
All these properties are modeled to have type `Position` which (see below) has *integer* `x` and `y`. Either the example or the model is wrong, please adjust one accordingly. (Maybe you have two types of "Position", one with integer coordinates and one without?)